src/server/Cypher.js

```diff
@@ -122,4 +133,8 @@ router.post("/", async (req, res) => {
   }
 });
 
+router.get("/result/:uuid", (req, res) => {
```

Comment:
If I understand correctly, this is actually used to get the progress information instead of the query result. So I would rename this to `progress`.